WalletWasabi.Fluent/Views/AddWallet/Create/ConfirmRecoveryWordsView.axaml

```diff
@@ -12,29 +12,95 @@
-                  <behaviors:FocusNextItemBehavior IsFocused="{Binding !IsConfirmed}" />
-                </i:Interaction.Behaviors>
-              </TextBox>
-            </DockPanel>
+                 EnableNext="True"
+                 NextContent="Continue"
+                 EnableSkip="{Binding IsSkipEnabled}"
+                 IsBusy="{Binding IsBusy}"
+                 ScrollViewer.VerticalScrollBarVisibility="Disabled">
+    <Grid x:Name="Layout">
+      <Grid.RowDefinitions>
+        <RowDefinition MinHeight="70" Height="*" />
+        <RowDefinition Height="16" />
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+        <RowDefinition Height="20" />
+      </Grid.RowDefinitions>
+
+      <c:RecoveryWordsControl Items="{Binding AvailableWords}">
+        <c:RecoveryWordsControl.Styles>
+          <Style Selector="StackPanel">
+            <Setter Property="HorizontalAlignment" Value="Center" />
+          </Style>
+          <Style Selector="ItemsControl">
+            <Setter Property="ItemsPanel">
+              <ItemsPanelTemplate>
+                <StackPanel Orientation="Horizontal" />
+              </ItemsPanelTemplate>
+            </Setter>
+          </Style>
+        </c:RecoveryWordsControl.Styles>
+        <c:RecoveryWordsControl.ItemTemplate>
+          <DataTemplate x:DataType="create:RecoveryWordViewModel">
+            <CheckBox IsChecked="{Binding IsSelected}"
+                      IsEnabled="{Binding IsEnabled}">
+              <CheckBox.Styles>
+                <Style Selector="CheckBox[IsChecked=True] /template/ c|TagControl">
+                  <Setter Property="Foreground" Value="{DynamicResource SystemAccentColor}" />
+                  <Setter Property="BorderBrush" Value="{DynamicResource SystemAccentColor}" />
+                </Style>
+                <Style Selector="CheckBox[IsEnabled=False] /template/ c|TagControl">
+                  <Setter Property="Foreground" Value="{DynamicResource ButtonForegroundDisabled}" />
+                  <Setter Property="BorderBrush" Value="{DynamicResource ButtonForegroundDisabled}" />
+                </Style>
+              </CheckBox.Styles>
+              <CheckBox.Template>
+                <ControlTemplate>
+                  <c:TagControl DataContext="{Binding Word}" Content="{Binding .}" ToolTip.Tip="{x:Null}" />
+                </ControlTemplate>
+              </CheckBox.Template>
+            </CheckBox>
           </DataTemplate>
-        </ItemsControl.ItemTemplate>
-      </ItemsControl>
-    </ScrollViewer>
+        </c:RecoveryWordsControl.ItemTemplate>
+      </c:RecoveryWordsControl>
+
+      <Panel Grid.Row="2">
+        <ItemsControl Items="{Binding ConfirmationWords}" VerticalAlignment="Center" HorizontalAlignment="Center" x:Name="ItemsControl">
+          <ItemsControl.Styles>
+            <Style Selector="TextBlock">
+              <Setter Property="FontSize" Value="{DynamicResource FontSizeH6}" />
```

Comment:
Set this locally rather than with styles, seems to be not necessary to use style here.